src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1145,29 +1145,23 @@ namespace WaterToAirHeatPumpSimple {
             state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).WaterMassFlowRate =
                 state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).DesignWaterMassFlowRate;
 
-            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate =
-                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                PsyRhoAirFnPbTdbW(state,
-                                  state.dataEnvrn->StdBaroPress,
-                                  state.dataLoopNodes->Node(AirInletNode).Temp,
-                                  state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                  RoutineName);
+            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate = state.dataLoopNodes->Node(AirInletNode).MassFlowRate;
```

Comment:
So now AirMassFlow rate is either the operating air flow rate (cont fan) or operating flow rate * PLR (cyc fan). This is correct but care should be used in the model. The coil model should calculate the full load result (i.e., full air flow at coil ON) and then average the outlet conditions for cont fan mode.

The DX coil does something like this:

    AirMassFlow = state.dataDXCoils->DXCoil(DXCoilNum).InletAirMassFlowRate
    if (FanOpMode == CycFanCycCoil) AirMassFlow /= PartLoadRatio;

So that the remaining calculations use full operating air flow rate.